v2_1/ws/rest/docs/4_8_metadata_queries.md

```diff
@@ -0,0 +1,150 @@
+
+### Examples
+
+* To retrieve version 1.0.0 of the Metadataset with id QUALITY_REPORT maintained by the ECB
+
+        http://ws-entry-point/metadata/metadataset/ECB/*/QUALITY_REPORT/1.0.0
+
+* To retrieve all the Metadatasets maintained by the ECB, without the reported metadata attributes
+
+        http://ws-entry-point/metadata/metadataset/ECB/?detail=allstubs
+
+## metadata/metadataflow
+### Purpose
+This resource enables clients to find metadatasets by the collection (metadataflow), optionally filtered by the metadata provider
+
+### Parameters
+
+#### Parameters used for identifying a resource
+
+The following parameters are used for identifying resources:
+
+Parameter | Type | Description
+--- | --- | ---
+metadataflowID | A string compliant with the SDMX _common:NestedNCNameIDType_ for Metadataflow | The id of the metdataflow the Metadata Set is reported against.  It is possible to set more than one id, using `,` as separator (e.g. MDF1,MDF2)
+flowAgencyID | A string compliant with the SDMX *common:NCNameIDType* | The agency maintaining the metadataflow for which metadata have been reported.
+flowID | A string compliant with the SDMX *common:IDType* | The id of the metadataflow for which metadata have been reported.
+flowVersion |  A string compliant with the *VersionType* defined in the SDMX Open API specification | The version of the metadataflow for which data have been reported.
+providerRef | A string identifying the metadata provider. The syntax is agency id, provider id, separated by a ",". For example: AGENCY_ID,PROVIDER_ID. In case the string only contains one out of these 2 elements, it is considered to be the provider id, i.e. all,PROVIDER_ID. | The provider of the data (or metadata) to be retrieved. If not supplied, the returned message will contain data (or metadata) provided by any provider. Its a common use case in SDMX-based web services that the provider id is sufficient to uniquely identify a data provider. Should this not be the case, the agency can be used, in conjunction with the provider id, in order to uniquely identify a data provider. The OR operator is supported using the + character. For example, the following value can be used to indicate that the metadataset should be provided by the Swiss National Bank (CH2) or Central Bank of Norway (NO2): CH2+NO2.
+
+The parameters mentioned above are specified using the following syntax:
+
+    protocol://ws-entry-point/metadata/metadataflow/flowRef/providerRef
```

Comment:
Should this not be `protocol://ws-entry-point/metadata/metadataflowID/flowAgencyID/flowID/flowVersion/providerRef` instead?

Reply:
correct, this is changed now, thanks